pycqed/instrument_drivers/meta_instrument/qubit_objects/CCL_Transmon.py

```diff
@@ -3680,6 +3707,116 @@ def measure_gst_1Q(self,
         a = ma2.GST_SingleQubit_DataExtraction(label='Single_qubit_GST')
         return a
 
+    def make_qubit_dac_arc(self, dac_values=None, polycoeffs=None, MC=None,
```

Comment:
What does the "make" prefix mean in the qubit object? Introducing a new prefix implies introducing a new concept. There is probably some good reason for this but I'm not sure what this is? 

It sounds more like this is something like measure flux arc. Also, we call it dac arc sometimes but this is stricltly speaking incorrect. DAC stands for Digitial Analog Conversion, or the DAC values of the current source. It is more sensible to call this the flux arc. 

Maybe the name `measure_flux_arc_spectroscopy(self, dac_values, polycoeffs, tracked=True)` is a better name?